lib/inject.js

```diff
@@ -85,10 +85,16 @@ function injectPa11y(window, options, done) {
 	}
 
 	function buildElementIdentifier(element) {
-		var identifier = (element.id ? '#' + element.id : element.tagName.toLowerCase());
+		if (element.id) {
+			return '#' + element.id;
```

Comment:
That line's been removed :) the new conditional is so that we don't need the ternary and we return early